Add embed command (#10)
Imageomics/pybioclip: README.md

## Command Line Usage
```

> bioclip predict [options] [IMAGE_FILE...]

Reviewer: Shouldn't the options be listed out here as well?

Reviewer 2: ```suggestion
bioclip predict [-h] [--format {table,csv}] [--output OUTPUT] [--rank {kingdom,phylum,class,order,family,genus,species}] [--k K] [--cls CLS] [--device DEVICE] image_file [image_file ...]
```
I'd second this since the shift to `argparse` requires `bioclip <predict/embed> -h` to get the options for either, while `bioclip -h` prints:
```console
usage: bioclip [-h] {predict,embed} ...

BioCLIP command line interface

options:
  -h, --help       show this help message and exit

commands:
  {predict,embed}
    predict        Use BioCLIP to generate predictions for image files.
    embed          Use BioCLIP to generate embeddings for image files.
```

bioclip embed [-h] [--device=DEVICE] [--output=OUTPUT] [IMAGE_FILE...]

Commands:
    predict            Use BioCLIP to generate predictions for image files.
    embed              Use BioCLIP to generate embeddings for image files.

Arguments:
  IMAGE_FILE           input image file

Options:
  -h --help

>   --format=FORMAT      format of the output (table or csv) [default: csv]

Reviewer: ```suggestion
  --format=FORMAT      format of the output (table or csv) for predict mode [default: csv]
```

  --rank=RANK          rank of the classification (kingdom, phylum, class, order, family, genus, species) [default: species] 
  --k=K                number of top predictions to show [default: 5]
  --cls=CLS            comma separated list of classes to predict, when specified the --rank and --k arguments are ignored [default: all]

>   --device=DEVICE      device to use for prediction (cpu or cuda or mps) [default: cpu]

Reviewer: ```suggestion
  --device=DEVICE      device to use matrix math (cpu or cuda or mps) [default: cpu]
```
